doc/getting_started/getting_started.rst

```diff
@@ -231,13 +231,23 @@ without the Zephyr SDK`_ for details.
    and extract it on your file system. This example assumes the compiler was
    extracted to: :file:`<user folder>/gcc-arm-none-eabi-5_3-2016q1/`.
 
+   On macOS, the toolchain can also be installed using brew::
+
+   # to tap the repository
+   $ brew tap osx-cross/arm
```

Comment:
the text box is really annoying to copy-paste into a terminal with the comments and the leading $, I'd like to see those removed.